src/app/src/test/java/com/cmput301f21t09/budgetprojectname/models/WeeklyHabitScheduleModelTest.java

```diff
@@ -45,19 +45,19 @@ public void testGetAllDays() {
         }
     }
 
-    /**
-     * Test for toMap method
-     */
-    @Test
-    public void testToMap() {
-        boolean[] days = {true, false, true, false, false, false, false};
-        for (int i = 0; i < days.length; i++) {
-            model.setDay(i, days[i]);
-        }
-        Map<String, Object> map = model.toMap();
-        Object innerMap = map.get("weekly");
-        assertNotNull(innerMap);
-    }
+//    /**
+//     * Test for toMap method
+//     */
+//    @Test
+//    public void testToMap() {
```

Comment:
not sure why this test is commented out

Reply:
Mac changes testToMap to testToList so this test not working anymore. I think I commented out when I was updating other test as this keep throw the error. I reverted it